content/en/docs/ops/ambient/getting-started/index.md

```diff
@@ -53,7 +53,7 @@ Follow these steps to get started with ambient:
 
     {{< text bash >}}
     $ kubectl get crd gateways.gateway.networking.k8s.io &> /dev/null || \
-      { kubectl kustomize "github.com/kubernetes-sigs/gateway-api/config/crd/experimental?ref=v0.6.1" | kubectl apply -f -; }
+      { kubectl kustomize "github.com/kubernetes-sigs/gateway-api/config/crd/experimental?ref=v0.7.1" | kubectl apply -f -; }
```

Comment:
Why are these docs using a hard coded version, instead of the latest version like all the other Istio docs?
```suggestion
      { kubectl kustomize "github.com/kubernetes-sigs/gateway-api/config/crd/experimental?ref={{< k8s_gateway_api_version >}}" | kubectl apply -f -; }
```

Reply:
My guess is because we want to reference a particular semver release rather than the commit SHA that currently exists. I can create another arg (e.g. k8s_gateway_api_tag) if that would be better?